<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TutorialMaker</class>
 <widget class="qMRMLWidget" name="TutorialMaker">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>339</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>409</width>
    <height>16777215</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinAndMaxSize</enum>
   </property>
   <item>
    <widget class="ctkCollapsibleButton" name="CollapsibleButtonTutorialMaking">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Tutorial Making</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QPushButton" name="pushButtonEdit">
        <property name="text">
         <string>Edit</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QPushButton" name="pushButtonLoad">
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="pushButtonSave">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QPushButton" name="pushButtonExportScreenshots">
        <property name="text">
         <string>Export Screenshots</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonFetchFromGithub">
     <property name="text">
      <string>Fetch From Github</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidgetTutorials"/>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonAnnotate">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Run and Annotate</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonTestPainter">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Test Translation</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonNewTutorial">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Create a new Tutorial</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonOpenAnnotator">
     <property name="text">
      <string>Open Annotator</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
